styles/globals.css

```diff
@@ -1,3 +1,8 @@
+@font-face {
+  font-family: "AlmondCream";
+  src: local("AlmondCream"), url(../public/AlmondCream.woff) format("woff");
```

Comment:
This works, but you'll have AlmondCream.woff two times. 
One time in your public folder `/AlmondCream.woff` and one in `/static/media/AlmondCream.219881f0.woff`.
I would move your fonts to `styles/fonts`.
An alternative is to change the url to access from public folder `url(/AlmondCream.woff)`.